openghg/standardise/_standardise.py

```diff
@@ -52,16 +61,22 @@ def standardise(
         pass
 
     with dclass(bucket=bucket) as dc:
-        result = dc.read_file(filepath=filepath, **kwargs)
-
+        if dataset is not None:
+            logger.info(
+                "The ability to pass dataset directly to the standardise function is currently availble for {data_type = ['surface'], source_format=['agage', 'openghg', 'co2_games']} "
```

Comment:
I think we probably want to avoid hard-coding this here as this is likely to change and we'd have to make sure this line stayed up-to-date. May be best to remove this logging message and just allow the individual parsers to raise the error (or could add a check on BaseStore for if the parse function beng called accepts `dataset`).